src/main.js

```diff
@@ -160,3 +162,24 @@ function buttonAction() {
     displayCard(ideaCards);
   }
 }
+
+function searchIdeas(event) {
```

Comment:
This function looks fantastic! Great work with those conditionals

Reply:
I agree with Kristi, this is awesome and I love the use of the logical "Or" operator. One thing to consider with expressions like `if (searchInput.value === '')` is that an empty string is considered a _falsey_ value, so this could be refactored as `if (!searchInput.value)`. [This article](https://j11y.io/javascript/truthy-falsey/#:~:text=There%20are%20only%20six%20falsey,%2C%20and%20false%20%2C%20of%20course.&text=For%20example%2C%200%20(zero),false)%20which%20is%20also%20truthy!) and many of the resources linked in the comments are great if you want to take a deeper dive into `truthy & falsey` land!

Reply:
Sorry, that link copied weird! Here is the full link: 

https://j11y.io/javascript/truthy-falsey/#:~:text=There%20are%20only%20six%20falsey,%2C%20and%20false%20%2C%20of%20course.&text=For%20example%2C%200%20(zero),false)%20which%20is%20also%20truthy!